ocaml/xapi/xapi_host_helpers.ml

```diff
@@ -457,6 +457,8 @@ module Configuration = struct
 
   let set_initiator_name iqn =
     let hostname = Unix.gethostname () in
+    (* CA-377454 - robustness, create dir if necessary *)
+    Unixext.mkdir_rec "/var/lock/sm/iscsiadm" 0o755 ;
```

Comment:
0o700 I think (the SM code looks like it is also a bit lax here so that needs to be addressed separately)